principles/index.html

```diff
@@ -83,6 +83,12 @@ <h3>Measurement</h3>
 
     <section>
       <h4>Measurement should be private for safe, widespread usage, but always be under user control</h4>
+
+      <p>Measurement and attribution are <a href="https://w3ctag.github.io/privacy-principles/#dfn-ancillary-use">ancillary uses</a> with functionality that may rely on broad usage. (Some forms of measurement are done with smaller, opt-in panels that typically provide identifiable data.) In order to be sufficiently private, measurement should not reveal <a href="https://w3ctag.github.io/privacy-principles/#dfn-data">personal data</a>, by, for example, ensuring measurement data is <a href="https://w3ctag.github.io/privacy-principles/#dfn-controlled-de-identified-data">controlled de-identified data</a>.</p>
+
+      <p>Because measurement and attribution involve all kinds of viewing advertisements and a variety of other actions, in a wide range of different contexts, relying on understanding of, expectations about and consent over cross-context recognition as a result of ad measurement would be inappropriate.</p>
```

Comment:
```suggestion
      <p>Measurement and attribution involve all kinds of viewing advertisements and a variety of other actions, in a wide range of different contexts. Relying on understanding of, expectations about, or consent over cross-context recognition as a result of ad measurement would be inappropriate.</p>
```

I really want an Oxford comma here :)